packages/gatsby/cache-dir/loader.js

```diff
@@ -529,6 +529,7 @@ export const publicLoader = {
   isPageNotFound: rawPath => instance.isPageNotFound(rawPath),
   hovering: rawPath => instance.hovering(rawPath),
   loadAppData: () => instance.loadAppData(),
+  getStaticQueryDb: () => Object.fromEntries(instance.staticQueryDb),
```

Comment:
Can we instead just have `staticQueryDb` as plain object and not Map to save on those conversions on each render? Also `Object.fromEntries` would need polyfilling possibly? It's quite new API

Reply:
Yeah, we'll need a polyfill. I agree that a plain object is probably a better idea. Map is better semantically and more consistent with the rest of the dbs though.

Reply:
Done

Reply:
Might have an adverse perf implication if there are many of these queries. I don't know if that number scales with the number of pages or just the number of source code files.

Reply:
Additionally, the returned object is now _shared_ by reference, where it was a fresh reference before. Not sure how relevant that is..?

Reply:
> I don't know if that number scales with the number of pages or just the number of source code files.

This scales with number of source files, not nodes/data

Reply:
@pvdz Great questions! 

Shared by reference is by design and should be fine. And as @pieh mentioned, `n` here will be the number of unique static queries and templates (and not pages built from these templates) so we should be fine.